src/dashboards/JWCC.vue

```diff
@@ -412,7 +412,7 @@ export default class JWCCDashboard extends Vue {
   public getCurrencyString = getCurrencyString;
 
   public async calculateSpendRateLineChartData(): Promise<void> {
-    const agencies = this.fundsSpentByServiceAgency.map(a => a.name);
+    const agencies = this.fundsSpentByAgency.map(a => a.name);
```

Comment:
In the screencap below, something is preventing name from being populated on the JWCC page.  I believe once that is resolved, there will be data for the charts on the JWCC page.

To get to the JWCC page, look at the 2nd screencap

console error on JWCC page
![image](https://user-images.githubusercontent.com/84856468/195081657-e389c209-9f36-4614-93d5-45f267f99542.png)

How to access the JWCC page
![image](https://user-images.githubusercontent.com/84856468/195081979-fb0cac34-a1d7-4dd0-a0ad-8dfd3186ef31.png)

Reply:
Fixed in https://github.com/dod-ccpo/atat-web-ui/pull/1055/commits/498704fdd2ff23bf95c606c326c551c50b5ea0a1 see comment for more details. In summary,
- old `Service/Agency` still lingering in `develop`, updated
- update request to get the `agency.title` for each agency